src/outlier_detection/auto_outliers.py

```diff
@@ -58,7 +56,7 @@ def filter_valid(df: pd.DataFrame, value_col: str) -> pd.DataFrame:
         pd.DataFrame: The filtered dataframe
     """
     sample_cond = df.selectiontype == "P"
-    status_cond = df.statusencoded.isin(["210", "211"])
+    status_cond = df["status"].isin(["Clear", "Clear - overridden"])
```

Comment:
This is unrelated to this feature. I am happy to pass it, but please let's have one feature/one branch/one PR.

Reply:
sorry! I did it as the unit tests were failing without this in 3.10

Reply:
Thanks Anne!